### ISSUE TITLE
Create Modern Fanatics set

### ISSUE THREAD

**Issue by dwvanstone** [buttonmen-dev/buttonmen#2867]
I'd like to propose that instead of having "20xx Fanatics" sets (one per year in which people submit fanatics) we have just one called "Modern Fanatics" that grows as people have their fanatics ready for submission.

**Comment by dwvanstone** [comment]
I have a current member who is almost ready to add their fanatic. Would this be the right path for us to have that user add their fanatic? Or should we have a "2023 Fanatics" with (possibly) a single button?

**Comment by cgolubi1** [comment]
I don't feel strongly.  I'd say create an issue for the player's fanatic once it's ready to test, and link to this and ask there.  We did lose track of this issue, but if we have an actual proposed fanatic, we'll try to play closer attention!

**Comment by dwvanstone** [comment]
I have changed my mind on the name of this set. I think it should be "Fanatics" rather than "Modern Fanatics." (With the theory that my dad's "Modern Chemistry" book from the 40's is just funny now.) 

I think having "Fanatics" and "Classic Fanatics" should be fine.

**Comment by irilyth** [comment]
As part of this, should we move the existing buttons in the "20xx Fanatics" sets into this one, and remove those sets? That sounds good to me, anyway.

**Comment by dwvanstone** [comment]
Yes - I propose moving all the "20xx Fanatics" sets into this new more generically named one.

**Comment by dwvanstone** [comment]
We had some discussion in the [group chat](http://buttonweavers.com/ui/forum.html#!threadId=258) about this and seemed to come to the conclusion that the name of this set should be simply "Fanatics". So that's a smidge of progress!

**Comment by dwvanstone** [comment]
I'd like to add the first new button to the new "Fanatics" set. 

Name: nycavri

Recipe:  (1) (3) (5) (14) Hs^(14)

Description:  
Described as "subtle and engaging" by Lawrence Block, when not beating people up nycavri is reading/writing novels, playing/designing games, and living/loving life. 
Art by Lauren Rogers

![Image](https://github.com/user-attachments/assets/f69c6efe-b8c2-44d3-9868-6c79b7f7f466)

**Comment by xmbrst** [comment]
I'd like to add an xmbrst fanatic to the new "Fanatics" set.

Name: xmbrst

Recipe: dk(1) dk(2) (13) (R) (R)!

Description: Xmbrst is an itinerant theremin player. He likes word machines, lime-green submarines, and beating people up.

![Image](https://github.com/user-attachments/assets/58e4be0d-0523-4e71-9095-505938adf86f)

**Comment by xmbrst** [comment]
Let me know if you need me to resize the image.

**Comment by xmbrst** [comment]
Dang -- got the recipe wrong on the image! Fixing...

**Comment by dwvanstone** [comment]
I know that there were games on the staging site with this recipe (such as https://staging.buttonweavers.com/ui/game.html?game=241). I'm not sure what you're asking here.

**Comment by dwvanstone** [comment]
Hmm...I don't know what a standard amount of testing is. I would believe that if a fan on our site has a favorite recipe they want to add to our site, even if it loses all the test games or wins all the test games, it should be accepted. It shouldn't break the site of course (like irilyth's fanatic currently does) and it shouldn't be offensive in image or text. Otherwise I think we should accept any new fanatic. We're not worrying about fairness of new buttons - they aren't TL.

**Comment by xmbrst** [comment]
Thanks all!. FYI, I played 17 games on staging with this recipe back in 2022: https://staging.buttonweavers.com/ui/history.html#!playerNameA=xmbrst&gameStartMax=2022-11-01&status=COMPLETE&numberOfResults=100
